code/modules/fishing/fish/types/rift.dm

```diff
@@ -0,0 +1,394 @@
+/obj/item/fish/starfish/chrystarfish
+	name = "chrystarfish"
+	fish_id = "chrystarfish"
+	desc = "This is what happens when a cosmostarfish sneaks into the bluespace compartment of a hyperspace engine. Very pointy and damaging - leading cause of spaceship explosions in 2554."
+	icon = 'icons/obj/aquarium/rift.dmi'
+	dedicated_in_aquarium_icon = 'icons/obj/aquarium/rift.dmi'
+	icon_state = "chrystarfish"
+	force = 12
+	sharpness = SHARP_POINTY
+	wound_bonus = -10
+	bare_wound_bonus = 15
+	armour_penetration = 6
+	demolition_mod = 1.2
+	throwforce = 11
+	throw_range = 8
+	throw_speed = 4
+	embed_type = /datum/embed_data/chrystarfish
+	attack_verb_continuous = list("stabs", "jabs")
+	attack_verb_simple = list("stab", "jab")
+	hitsound = SFX_SHATTER
+	pickup_sound = 'sound/items/handling/materials/glass_pick_up.ogg'
+	drop_sound = 'sound/items/handling/materials/glass_drop.ogg'
+
+	sprite_width = 7
+	sprite_height = 9
+
+	average_size = 40
+	average_weight = 1500
+	food = /datum/reagent/bluespace
+	feeding_frequency = 10 MINUTES
+	health = 50
```

Comment:
You know, I wasn't really against the 300 health because that way it lasts long enough to be weaponized properly, also you really need to add that variable to control the damage dealt to the fish when bitten while it's still alive, otherwise the flinch_on_eat proc won't be all that useful if it dies in one bite anyway.

Reply:
the fish is allphibious so it dying from exposure isnt a concern. i lowered health (because it should be fragile-ish see sounsd & embed) but upped weight so now it doesnt die in a single bite anymore

Reply:
Okay dokay